packages/inventory/src/containers/ListView/index.tsx

```diff
@@ -0,0 +1,173 @@
+import React, { useEffect, useState } from 'react';
+import { Row, PageHeader, Col, Button, Table } from 'antd';
+import { OpenSRPService } from '@opensrp/react-utils';
```

Comment:
Why not import from `@opensrp/server-service`?

Reply:
@kelvin-muchiri react-utils provides a wrapper service class that adds accesstoken to the opensrp/server-service server class